tf/package.xml

```diff
@@ -41,11 +40,13 @@ any desired point in time.
   <run_depend>roswtf</run_depend>
   <run_depend>sensor_msgs</run_depend>
   <run_depend>std_msgs</run_depend>
-  <run_depend version_gte="0.4.5">tf2</run_depend>
   <run_depend version_gte="0.4.5">tf2_ros</run_depend>
 
+  <test_depend>rostest</test_depend>
+  <test_depend>rosunit</test_depend>
+
   <export>
-    <roswtf plugin="tf.tfwtf" />
+    <!-- <roswtf plugin="tf.tfwtf" /> -->
```

Comment:
tfwtf is written for tf1 (e.g. it explicitly subscribes to `/tf`), so IMHO this functionality should be implemented by tf2 instead.